api/apps/api/src/modules/scenarios/scenarios.controller.ts

```diff
@@ -1232,7 +1232,7 @@ export class ScenariosController {
   async findLocksForScenariosWithinParentProject(
     @Param('scenarioId', ParseUUIDPipe) scenarioId: string,
     @Req() req: RequestWithAuthenticatedUser,
-  ): Promise<null | ScenarioLockResultSingular> {
+  ): Promise<{ data: null } | ScenarioLockResultSingular> {
```

Comment:
mh, what about adding to `ScenarioLockResultSingular` and `ScenarioLockResultPlural` the option for data to be undefined, instead? it's a small thing, but I'd try to avoid ad-hoc inline types if possible.